ProblemsSolved.js

```diff
@@ -0,0 +1,76 @@
+function ProblemsSolved() 
+{
+}
+
+ProblemsSolved.prototype.SumOfEvenNumberInArray = function(array)
+{
+    var sum = 0;
+    for(i=0; i < array.length; i++)
```

Comment:
Because I used C# to implement problem first and after that I converted the code to javascript, I missed to exchange the type of variable (in that case : int i ) with "var" from javascript. But was working and  I have not observed the mistake. Ugly mistake :)